Sources/PointFree/Stripe.swift

```diff
@@ -468,9 +466,9 @@ extension Tagged where Tag == Stripe.Plan, RawValue == String {
 }
 
 func cancelSubscription(id: Stripe.Subscription.Id) -> DecodableRequest<Stripe.Subscription> {
-  return stripeRequest(
-    "subscriptions/" + id.rawValue + "?expand[]=customer", .delete(["at_period_end": "true"])
-  )
+  return stripeRequest("subscriptions/" + id.rawValue + "?expand[]=customer", .post([
+    "cancel_at_period_end": "true"
```

Comment:
Cancellation endpoint now cancels things _immediately_. Need to hit the update endpoint for cancellation at period end.